tailcfg/tailcfg.go

```diff
@@ -1347,9 +1342,11 @@ type MapResponse struct {
 	// ControlTime, if non-zero, is the current timestamp according to the control server.
 	ControlTime *time.Time `json:",omitempty"`
 
-	// TKA, if non-nil, describes updates for the local instance of the
-	// tailnet key authority.
-	TKA *TKAMapResponse `json:",omitempty"`
+	// TKAInfo, if non-nil, describes the control plane's view of tailnet
+	// key authority state.
+	//
+	// A nil TKAInfo indicates no changes from before.
```

Comment:
You forgot half my request. What's an initial nil mean? Does it mean Enabled==false, Head==""?

Reply:
A nil `TKAInfo` means don't touch the local TKA state, rather than declaring that tka should or shouldnt be enabled or anything like that (that would mean that not sending TKAInfo because it hasnt changed has the side effect of reaching out to get the disablement secret, regardless of if its actually disabled).

Ive updated the comment.